config/crd/bases/self-node-remediation.medik8s.io_selfnoderemediations.yaml

```diff
@@ -82,9 +82,10 @@ spec:
                     description: Spec defines the behavior of a node. https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#spec-and-status
                     properties:
                       configSource:
-                        description: If specified, the source to get node configuration
-                          from The DynamicKubeletConfig feature gate must be enabled
-                          for the Kubelet to use this field
+                        description: 'Deprecated: Previously used to specify the source
```

Comment:
Is there any reason why the whole sentence/block has an apostrophe in the beginning and in the end?
Normally we don't use it

Reply:
Same as comment above